issue.go

```diff
@@ -16,14 +16,20 @@ import (
 
 type IssueQueryProvider interface {
 	ProvideIssueQueryString() string
+	ProvideDefaultProject() string
 }
 
 type IssueOptions struct {
-	Fields        []string `json:"fields,omitempty" yaml:"fields,omitempty"`
-	Expand        []string `json:"expand,omitempty" yaml:"expand,omitempty"`
-	Properties    []string `json:"properties,omitempty" yaml:"properties,omitempty"`
-	FieldsByKeys  bool     `json:"fieldsByKeys,omitempty" yaml:"fieldsByKeys,omitempty"`
-	UpdateHistory bool     `json:"updateHistory,omitempty" yaml:"updateHistory,omitempty"`
+	Fields	    []string `json:"fields,omitempty" yaml:"fields,omitempty"`
```

Comment:
please run `gofmt` against this, the previous formatting for these lines was correct.